### PR TITLE
install_rotate_1.0

### PR DESCRIPTION
**Pull request by jayramr**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by jayramr** [comment]
@BiocondaBot please update

**Comment by jayramr** [comment]
@BiocondaBot please fetch artifacts

File: recipes/rotate/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+  host:
+    - zlib
+  run:
+    - zlib
+    - make
```
[the remark this answers is not in the document]
Comment (martin-g) [reply recipes/rotate/meta.yaml:25]: @jayramr is make really needed as a runtime dependency?

[the remark this answers is not in the document]
Comment (jayramr) [reply recipes/rotate/meta.yaml:25]: sorry that was an error, make only needed for the build time. I will remove it and commit.

**Comment by jayramr** [comment]
@BiocondaBot please add label

File: recipes/rotate/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    - {{ compiler('c') }}
+  host:
+    - zlib
+  run:
+    - zlib
+
+
+test:
+  commands:
+    - rotate 2>&1 | grep can
```
Comment (bgruening) [inline recipes/rotate/meta.yaml:29]: why "grep can" ?

File: recipes/rotate/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    - {{ pin_subpackage("rotate", max_pin="x") }}
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+  host:
+    - zlib
+  run:
+    - zlib
```
Comment (bgruening) [inline recipes/rotate/meta.yaml:24]:
```suggestion
```

File: recipes/rotate/meta.yaml (hunk 2 above)
Comment (jayramr) [reply recipes/rotate/meta.yaml:29]: hi @bgruening for this package, the help menu shows the whole output. To simply the output, i just did a grep. If this is not a recommended practice then , we can keep below as it as. Please advise 

```
rotate
rotate [options] <input fasta filename>
  -x <pos>		position to become new start [0]
  -rc               reverse complement after rotating [false]
  -s <string>       rotate to start of search string
  -m <n_mismatch>   maximum number of mismatches allowed in search [0]
  -o <outfilename>  output file [standard output]
use either -x and optionally -rc, or -s
use filename '-' for standard input
can read .fa.gz files as well as .fa files
```

**Review by mencian — APPROVED** [review]

**Pull request merged**